cmd/fleet-controller/hibernate.go

```diff
@@ -93,6 +93,8 @@ var hibernate = &cobra.Command{
 		logger.Infof("Calculating hibernate actions on %d stable installations", len(installations))
 		var errorSkipCount, maxUserSkipCount int
 		var installationsToHibernate []*cmodel.InstallationDTO
+		creationTimestampCutoff := (time.Now().UnixNano() / int64(time.Millisecond)) - (int64(days) * 24 * int64(time.Millisecond))
```

Comment:
I believe this calculation is a little off.
Given `days = 1` the `int64(days) * 24 * int64(time.Millisecond)` will result in `24000000` whereas one day is `86400000` milliseconds.
```suggestion
		creationTimestampCutoff := (time.Now().UnixNano() / int64(time.Millisecond)) - (int64(days) * 24 * int64(time.Hour / time.Millisecond))
```

Reply:
Yep, you are totally right. I had tested this and forgot to put it back. Thanks for catching it.